### PR TITLE
Fix archived repos

### PR DESCRIPTION
**Pull request by scoopex**

### CODE REVIEW

File: orgs/SovereignCloudStack/repositories/zuul_config.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ zuul_config:
   homepage: 'https://zuul.scs.community/'
   topics:
     - zuul
-  archived: false
```
Comment (gtema) [inline orgs/SovereignCloudStack/repositories/zuul_config.yml:8]: I see absolutely no point in archiving this repo. It doesn't have even historically any value. It should be just dropped.

File: orgs/SovereignCloudStack/repositories/zuul_sandbox.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ zuul_sandbox:
   homepage: 'https://zuul.scs.community'
   topics:
     - zuul
-  archived: false
+  archived: true
```
Comment (gtema) [inline orgs/SovereignCloudStack/repositories/zuul_sandbox.yml:8]: absolutely no useful content. Just drop it

File: orgs/SovereignCloudStack/repositories/zuul_deployment.yml [hunk 3]
```diff
@@ -5,7 +5,7 @@ zuul_deployment:
   homepage: 'https://zuul.scs.community/'
   topics:
     - zuul
-  archived: false
+  archived: true
```
Comment (gtema) [inline orgs/SovereignCloudStack/repositories/zuul_deployment.yml:8]: There are more "much more useful" demos on Zuul deployment. No value keeping it archived in my eyes

File: orgs/SovereignCloudStack/repositories/testbed-gx-k8s.yml [hunk 4]
```diff
@@ -6,7 +6,7 @@ testbed-gx-k8s:
   topics:
     - kubernetes
     - gaia-x
-  archived: false
+  archived: true
```
Comment (gtema) [inline orgs/SovereignCloudStack/repositories/testbed-gx-k8s.yml:9]: there is no content besides GH actions. Drop it

File: orgs/SovereignCloudStack/repositories/testbed-gx-scs.yml [hunk 5]
```diff
@@ -6,7 +6,7 @@ testbed-gx-scs:
   topics:
     - openstack-environment
     - gaia-x
-  archived: false
+  archived: true
```
Comment (gtema) [inline orgs/SovereignCloudStack/repositories/testbed-gx-scs.yml:9]: only readme file with 15 dummy lines. Drop it

File: orgs/SovereignCloudStack/repositories/slides.yml [hunk 6]
```diff
@@ -5,7 +5,7 @@ slides:
   homepage: 'https://scs.community/'
   topics:
     - community
-  archived: false
+  archived: true
```
Comment (gtema) [inline orgs/SovereignCloudStack/repositories/slides.yml:8]: a repo with single pdf - if that slidedeck is useful in any regard I would rather move it somewhere else and drop the repo. Historically this repo makes no sense

File: orgs/SovereignCloudStack/repositories/ort-docker.yml [hunk 7]
```diff
@@ -5,7 +5,7 @@ ort-docker:
   homepage: 'https://scs.community/'
   topics:
     - oss-review-toolkit
-  archived: false
+  archived: true
```
Comment (gtema) [inline orgs/SovereignCloudStack/repositories/ort-docker.yml:8]: useless repo which is itself a fork. Drop it

File: orgs/SovereignCloudStack/repositories/openstack-vyos-image.yml [hunk 8]
```diff
@@ -7,7 +7,7 @@ openstack-vyos-image:
     - openstack
     - openstack-environment
     - openstack-glance
-  archived: false
+  archived: true
```
Comment (gtema) [inline orgs/SovereignCloudStack/repositories/openstack-vyos-image.yml:10]: no content useful in any sense. drop

File: orgs/SovereignCloudStack/repositories/k8s-operator-scylla.yml [hunk 9]
```diff
@@ -5,7 +5,7 @@ k8s-operator-scylla:
   homepage: 'https://scs.community/'
   topics:
     - k8s
-  archived: false
+  archived: true
```
Comment (gtema) [inline orgs/SovereignCloudStack/repositories/k8s-operator-scylla.yml:8]: no useful content, no useful docs - useless

File: orgs/SovereignCloudStack/repositories/k8s-open-policy-agent.yml [hunk 10]
```diff
@@ -5,7 +5,7 @@ k8s-open-policy-agent:
   homepage: 'https://scs.community/'
   topics:
     - k8s
-  archived: false
+  archived: true
```
Comment (gtema) [inline orgs/SovereignCloudStack/repositories/k8s-open-policy-agent.yml:8]: no useful content, broken readme - drop it

File: orgs/SovereignCloudStack/repositories/k8s-harbor-manual.yml [hunk 11]
```diff
@@ -5,7 +5,7 @@ k8s-harbor-manual:
   homepage: 'https://scs.community/'
   topics:
     - k8s
-  archived: false
+  archived: true
```
Comment (gtema) [inline orgs/SovereignCloudStack/repositories/k8s-harbor-manual.yml:8]: absolutely useless

File: orgs/SovereignCloudStack/repositories/infrastructure.yml [hunk 12]
```diff
@@ -4,7 +4,7 @@ infrastructure:
   description: 'This repository should contain all automation manifests used to deploy central SCS provided infrastructure'
   homepage: 'https://scs.community/'
   topics: []
-  archived: false
+  archived: true
```
Comment (gtema) [inline orgs/SovereignCloudStack/repositories/infrastructure.yml:7]: no content - drop

File: orgs/SovereignCloudStack/repositories/gx-scs-identity-provider.yml [hunk 13]
```diff
@@ -5,7 +5,7 @@ gx-scs-identity-provider:
   homepage: 'https://scs.community/'
   topics:
     - gaia-x
-  archived: false
+  archived: true
```
Comment (gtema) [inline orgs/SovereignCloudStack/repositories/gx-scs-identity-provider.yml:8]: I do not see the content as useful - it looks lie it was created to be a template but never became such

File: check_consistency.py [hunk 14]
```diff
@@ -30,7 +30,9 @@ def check_repos(gh: Github) -> int:
             with open(os.path.join(repositories_dir, filename)) as f:
                 yaml_data = yaml.safe_load(f)
                 if yaml_data is not None:
-                    defined_repos.add(list(yaml_data.keys())[0])
+                    for r_name in yaml_data.keys():
+                        if "archived" in yaml_data[r_name] and not yaml_data[r_name]["archived"]:
```
Comment (gtema) [inline check_consistency.py:34]: could be just `if not yaml_data[r_name].get("archived", False): ...`

**Review by gtema — CHANGES_REQUESTED** [review]

**Comment by gtema** [comment]
in general I see no value in archiving what never become useful or not even received any content

File: orgs/SovereignCloudStack/repositories/zuul_config.yml (hunk 1 above)
Comment (scoopex) [reply orgs/SovereignCloudStack/repositories/zuul_config.yml:8]: Created a local backup on my workstation, deleted it, removed id from the config.

File: orgs/SovereignCloudStack/repositories/zuul_sandbox.yml (hunk 2 above)
Comment (scoopex) [reply orgs/SovereignCloudStack/repositories/zuul_sandbox.yml:8]: Created a local backup on my workstation, deleted it, removed id from the config.

File: orgs/SovereignCloudStack/repositories/zuul_deployment.yml (hunk 3 above)
Comment (scoopex) [reply orgs/SovereignCloudStack/repositories/zuul_deployment.yml:8]: Created a local backup on my workstation, deleted it, removed id from the config.

File: orgs/SovereignCloudStack/repositories/testbed-gx-scs.yml (hunk 5 above)
Comment (scoopex) [reply orgs/SovereignCloudStack/repositories/testbed-gx-scs.yml:9]: Created a local backup on my workstation, deleted it, removed id from the config.

File: orgs/SovereignCloudStack/repositories/k8s-open-policy-agent.yml (hunk 10 above)
Comment (scoopex) [reply orgs/SovereignCloudStack/repositories/k8s-open-policy-agent.yml:8]: Created a local backup on my workstation, deleted it, removed id from the config.

File: orgs/SovereignCloudStack/repositories/gx-scs-identity-provider.yml (hunk 13 above)
Comment (scoopex) [reply orgs/SovereignCloudStack/repositories/gx-scs-identity-provider.yml:8]: Created a local backup on my workstation, deleted it, removed id from the config.

File: orgs/SovereignCloudStack/repositories/slides.yml (hunk 6 above)
Comment (scoopex) [reply orgs/SovereignCloudStack/repositories/slides.yml:8]: Created a local backup on my workstation, deleted it, removed id from the config.

File: orgs/SovereignCloudStack/repositories/ort-docker.yml (hunk 7 above)
Comment (scoopex) [reply orgs/SovereignCloudStack/repositories/ort-docker.yml:8]: Created a local backup on my workstation, deleted it, removed id from the config.

File: orgs/SovereignCloudStack/repositories/k8s-operator-scylla.yml (hunk 9 above)
Comment (scoopex) [reply orgs/SovereignCloudStack/repositories/k8s-operator-scylla.yml:8]: Created a local backup on my workstation, deleted it, removed id from the config.

File: check_consistency.py (hunk 14 above)
Comment (scoopex) [reply check_consistency.py:34]: That's true, Tnx :-)

**Review by gtema — APPROVED** [review]

File: orgs/SovereignCloudStack/repositories/testbed-gx-k8s.yml (hunk 4 above)
Comment (scoopex) [reply orgs/SovereignCloudStack/repositories/testbed-gx-k8s.yml]: Created a local backup on my workstation, deleted it, removed id from the config.

**Pull request merged**